torchvision/utils.py

```diff
@@ -30,7 +30,7 @@ def make_grid(
             The final grid size is ``(B / nrow, nrow)``. Default: ``8``.
         padding (int, optional): amount of padding. Default: ``2``.
         normalize (bool, optional): If True, shift the image to the range (0, 1),
-            by the min and max values specified by :attr:`range`. Default: ``False``.
+            by the min and max values specified by :attr:`value_range`. Default: ``False``.
```

Comment:
It was already her, but the `:attr:` is not needed here (and not even correct actually)

```suggestion
            by the min and max values specified by ``value_range``. Default: ``False``.
```